frontend/src/component/PublicProfile/CustomFormManager.styl

```diff
@@ -0,0 +1,29 @@
+@import "~tracim_frontend_lib/src/css/Variable.styl"
+
+.CustomFormManager
+  position relative
+  &__title
+    margin-bottom 15px
+    max-width calc(100% - 100px) // CH - INFO - 20200126 - 100px is the validate button width in english
```

Comment:
I think "Modifier" in french is bigger

Reply:
yes but "Validate" in english is the longest

Reply:
no, "Validate" is longer by 1px. Close call though !